.travis.yml

```diff
@@ -8,4 +8,4 @@ before_install:
 
 script:
   - docker build -t dap_testing -f Dockerfile.testing .
-  - docker run --rm --name dap_testing -it -e DAP_EXECUTABLE=dap dap_testing /bin/bash -l -c "rvm use 2.4.5 && gem build dap && gem install dap*.gem && bundle exec rspec spec && find /opt/bats_testing -name \*.bats | grep -v test/test_helper/ | xargs -n1 bats"
+  - docker run --rm --name dap_testing -it -e DAP_EXECUTABLE=dap dap_testing /bin/bash -l -c "rvm use 2.7.6 && gem build dap && gem install dap*.gem && bundle exec rspec spec && find /opt/bats_testing -name \*.bats | grep -v test/test_helper/ | xargs -n1 bats"
```

Comment:
Instead of Travis consider moving this to github actions and completely removing `.travis.yml` as TravisCI is no longer offering credits for open-source test execution.

Reply:
That's a great point @jmartin-r7. This change was a find and replace and I had not taken a step back since we are no longer getting TravisCI builds. I'll remove the file as a part of this PR and put up another for the Github Actions.